packages/web/index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
```

Comment:
Anything of note different in this file?

Reply:
Yes, removed `PUBLIC_URL` because vite automatically creates paths that reference the assets folder. Also added the `window.Buffer` and `window.process` stuff here instead of index.jsx because it needed to happen earlier at runtime (solana was breaking)